bat/tests/09-ister-config/run.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "Build image using default ister config" {
+  mixer $MIXARGS init --clear-version $CLRVER --mix-version 10
+  mixer $MIXARGS bundle add editors
+
+  # `build image` should fail because the prerequisite commands are not executed.
+  # This is intentional in order to reduce test execution time.
+  # The goal is to unit test the creation of ister config file with relevant bundles.
+  run sudo mixer $MIXARGS build image --native
+  [[ "$status" -eq 1 ]]
+  [[ "$output" =~ "release-image-config.json not found" ]]
+  [[ "$output" =~ "Copying image template" ]]
+  [[ "$output" =~ "Updating image bundle list based on mixbundles" ]]
+}
+
+@test "Update ister config file with mix bundle list" {
+  run cat release-image-config.json
+  [[ "$status" -eq 0 ]]
```

Comment:
optional: the above two lines can be replaced with `cat release-image-config.json`

Reply:
I'm executing it with the `run` command so that the output can be accessed using  `$output`

Reply:
Okay, thanks for clarifying. I don't have strong opinions either way, but it seems cleaner to me to rewrite this test to use `grep` instead, avoiding any use of `$status` and `$output`. Something like:

```
config=release-image-config
[ -f $config ]
for b in os-core os-core-update kernel-native bootloader editors; do
  grep '"'$b'"' $config
done
```

Reply:
I'll go 1 step further and say 
`bundles="os-core os-core-update kernel-native bootloader editors"

for b in $bundles; do
  echo '"'$b'"'
done`

is a little clearer too.